js_adapter/js_class_adapter.ts

```diff
@@ -384,7 +384,10 @@ export class Decorators {
             let type: Type;
 
             // get template type
-            if (typeof params[0] == "string") type = Type.get(params[0].replace(/^\</,'').replace(/\>$/,''))
+            if (typeof params[0] == "string") {
```

Comment:
Duplicate code for that - Can we make this one reusable?

Reply:
Okay, I added a `normalizeType` that also distinguished between the `ext` and `std` namespace: For new type definitions (sync classes) the default namespace is `ext`, for `@type` decorators, it's `std`:
```ts
@sync("MyPerson") class Person { // type: <ext:MyPerson>
    @type("text(50)") @property name = "Name" // type: <std:text(50)>
}
```

Additionally, type parameters are only allowed in `@type` decorators, because they don't make sense with @sync